python/py-dqsegdb2/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-dqsegdb2
+version             1.0.0
+
+categories-append   science
+maintainers         {ligo.org:duncan.macleod @duncanmmacleod} openmaintainer
+
+platforms           darwin
+license             GPL-3
+
+description         Simplified python interface to DQSEGDB
+long_description    dqsegdb2 is a simplified Python implementation of the \
+                    DQSEGDB API as defined in LIGO-T1300625. \
+                    This package only provides a query interface for GET \
+                    bequests, any users wishing to make POST requests \
+                    should refer to the official 'dqsegdb' port.
+homepage            https://dqsegdb2.readthedocs.io
+
+master_sites        pypi:d/dqsegdb2
+distname            dqsegdb2-${version}
+checksums           rmd160 dc1275e619fb7bc9713903bde44a9a3f557fe936 \
+                    sha256 828a1afd0af31b4fda2bba3427836aa5cf7fb751b636140ba6e8c48d1e6aac34 \
+                    size   17980
+
+python.versions     27 36 37
+python.default_version  36
+
+if {${name} ne ${subport}} {
+    depends_lib-append  port:py${python.version}-gwdatafind \
+                        port:py${python.version}-ligo-segments
+
+    depends_build-append port:py${python.version}-setuptools
+
+    livecheck.type      none
+} else {
+    livecheck.type      pypi
+}
```

Comment:
FYI, I believe that it isn't actually necessary to say 

```
else {
    livecheck.type      pypi
}
```

because that is the default. @reneeotten, can you tell me if that's right?

Reply:
true, you don't need it as it's the default when you use ```pypi``` as ```master_site```. As a minor side note: the whitespace for the ```depends_build-append``` doesn't follow the rules.

Reply:
@reneeotten, can you remind me of the rules (or where to find them)?

Reply:
it's probably somewhere in the guide... but also specified in the modeline: spaces should be multiples of four. So the second element of a line should start at 21 (as in the first part of your Portfile), 25 (in the if-block); for your `depends_build-append` it starts at 26. You can either add three spaces, or break the line and align it with the statements before (my preference).